loginpage/js/loginpage.js

```diff
@@ -1,34 +1,37 @@
-$(document).ready(function()
-{
-	$.ajax(	"/api/login.json", {
-	    data: { checkLogin: true },
-		dataType: "json",
-		success: function (response) {
-			if(response.loggedIn){
-				$("#status-box").text(response.message);
-				$("#status-box").addClass("error");
-				$("#loginForm").addClass("hidden");
-				$("#logoutForm").removeClass("hidden");
-			}
-		},
-		error: function (xhr, ajaxOptions, thrownError) {
-			$("#status-box").text(thrownError);
-			$("#status-box").addClass("error");
-		},
-	});
+$(document).ready(function () {
+    $.ajax( "/api/login.json", {
+        data: { checkLogin: true },
+        dataType: "json",
+        success: function (response) {
+            if(response.loggedIn) {
+                $("#status-box").text(response.message);
+                $("#status-box").addClass("error");
+                $("#loginForm").addClass("hidden");
+                $("#logoutForm").removeClass("hidden");
+            }
+        },
+        error: function (xhr, ajaxOptions, thrownError) {
+            $("#status-box").text(thrownError);
+            $("#status-box").addClass("error");
+        },
+    });
 
-    function setRemember(){
-        if($("#remember").is(":checked")){
+    function setRemember()
+    {
+        if($("#remember").is(":checked")) {
             $("#remember_hidden").val("cookie");
         }
         else{
             $("#remember_hidden").val("session");
         }
     }
-    $("#remember").click(function(){setRemember();});
+    $("#remember").click(function ()
+        {
```

Comment:
Opening brace should be on the same line as the declaration

*PHPCodeSnifferBear, severity NORMAL, section `js`.*